game/templates/game/avatar.html

```
@@ -0,0 +1,191 @@
+                    </div>
+                </label>
+
+                <input type="file"
+                       name="avatar"
+                       id="id_avatar"
+                       accept="image/png,image/jpeg,image/webp"
+                       class="file-input"
+                       aria-label="Choose profile picture">
+
+                <div id="previewNewWrap" class="preview-new-wrap" style="display:none;">
+                    <img id="previewNewImg" class="preview-new-img" src="" alt="Preview of selected image">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)